.eslintrc.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
     '@typescript-eslint/no-unsafe-assignment': 0,
     '@typescript-eslint/no-unsafe-call': 0,
     '@typescript-eslint/unbound-method': 0,
+    '@typescript-eslint/restrict-template-expressions': 0,
+    '@typescript-eslint/no-unnecessary-type-assertion': 0,
```

Comment:
how come? should this be added to our main eslint config?

Reply:
issues with the double package.json set up, need to revert to running eslint on result of a webpack output